ol-keycloak/ol-spi/src/main/java/edu/mit/keycloak/admin/ui/OpenAdminUiTab.java

```diff
@@ -0,0 +1,85 @@
+package edu.mit.keycloak.admin.ui;
+
+import org.keycloak.Config;
+import org.keycloak.component.ComponentModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.KeycloakSessionFactory;
+import org.keycloak.models.RealmModel;
+import org.keycloak.provider.ProviderConfigProperty;
+import org.keycloak.provider.ProviderConfigurationBuilder;
+import org.keycloak.services.ui.extend.UiTabProvider;
+import org.keycloak.services.ui.extend.UiTabProviderFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.jboss.logging.Logger;
+
+import edu.mit.keycloak.util.OpenAttribute;
+/**
+ * Admin UI Tab for providing additional realm-level settings for MIT Open.
+ *
+ * These are things that don't fit well under different scopes like clients.
+ *
+ */
+public class OpenAdminUiTab implements UiTabProvider, UiTabProviderFactory<ComponentModel> {
```

Comment:
I would prepend "Ol" to this class name in order to more easily distinguish it against OOTB.